### ISSUE TITLE
Question about module architecture best practices....

### ISSUE THREAD

**Issue by natiki** [abpframework/abp#583]
Hi,

On https://abp.io/documents/abp/latest/Best-Practices/Module-Architecture it has "Do create a separated Visual Studio solution for every module." Isn't that supposed to be create a separate Visual Studio **project** for each module?

**Comment by hikalkan** [comment]
No, solution. This is for business modules (like [blogging module](https://github.com/abpframework/abp/tree/master/modules/blogging)) rather than a standard/core module (like [EF core intergation module](https://github.com/abpframework/abp/tree/master/framework/src/Volo.Abp.EntityFrameworkCore)).

A business module has already [9+ projects](https://abp.io/documents/abp/latest/Best-Practices/Module-Architecture#layers-packages) inside. Actually, every project inside it is an AbpModule.

But I should make the difference clearer between a project with AbpModule class and a business module which has multiple sub AbpModules.
